source/monitoring.txt

```diff
@@ -0,0 +1,383 @@
+==================
+Cluster Monitoring
+==================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to set up and configure **monitoring** in the
+MongoDB Kotlin driver.
+
+Monitoring is the process of getting information about the activities a running
+program performs for use in an application or an application performance
+management library.
+
+Monitoring the MongoDB Kotlin driver lets you understand the
+driver's resource usage and performance, and can help you make informed
+decisions when designing and debugging your application.
```

Comment:
S: since it's the application you're monitoring by using the driver, rather than monitoring the driver itself (i think):
```suggestion
You can use the {+driver-short+} to monitor cluster, driver command, and connection pool events. These features help you make informed
decisions when designing and debugging your application.
```